openpype/modules/deadline/repository/custom/plugins/GlobalJobPreLoad.py

```diff
@@ -385,6 +385,13 @@ def inject_openpype_environment(deadlinePlugin):
         for key, value in contents.items():
             deadlinePlugin.SetProcessEnvironmentVariable(key, value)
 
+        if "PATH" in contents:
+            # Set os.environ[PATH] so studio settings' path entries
+            # can be used to define search path for executables.
+            PATH = contents["PATH"]
+            print(f">>> Setting 'PATH' Environment to: {PATH}")
+            os.environ["PATH"] = PATH
```

Comment:
You missed this change for both :)
```suggestion
            print(f">>> Setting 'PATH' Environment to: {PATH}")
            os.environ["PATH"] = contents["PATH"]
```

Reply:
Ah wait, you are right - you're using the var for printing too.

This could work
```python
print(f">>> Setting 'PATH' Environment to: {contents['PATH']}")
```
But also this:
```python
print(">>> Setting 'PATH' Environment to: {[PATH]}".format(contents))
```
Which I think looks nicer but may be confusing to some.

Otherwise of course this is still an option too:
```python
print(">>> Setting 'PATH' Environment to: {}".format(contents["PATH"])
```

Anyway, since you're using the variable twice I'm fine with keeping it like you have it as well - however I feel like we should at least tweak the variable name to not be uppercased.

Reply:
I agree. It shouldn't be uppercase, which because it's not a constant. 
And, it shouldn't be lowercase either because there's a variable named `path` in that file

Reply:
> Would it be fine if I just used contents["PATH"] twice ?

Yes, just as fine.